posthog/temporal/common/worker.py

```diff
@@ -39,6 +40,8 @@ async def start_worker(
         workflow_runner=UnsandboxedWorkflowRunner(),
         graceful_shutdown_timeout=timedelta(minutes=5),
         interceptors=[SentryInterceptor()],
+        activity_executor=ThreadPoolExecutor(max_workers=50),
+        max_concurrent_activities=50,
```

Comment:
50 is temporal's default but I think we should really be able to bump this up :thinking: Anyways, we can experiment later.